docs/topics/dynamic-content.rst

```diff
@@ -85,6 +85,12 @@ It might be enough to yield a :class:`~scrapy.http.Request` with the same HTTP
 method and URL. However, you may also need to reproduce the body, headers and
 form parameters (see :class:`~scrapy.http.FormRequest`) of that request.
 
+As all the major browsers allow to export the requests in `cURL
+<https://curl.haxx.se/>`_ format, scrapy incorporates the method
+:meth:`~scrapy.http.Request.from_curl()` to generate an equivalent
+:class:`~scrapy.http.Request` from a cURL command. To get more information
+visit :ref:`the network tool section <topics-network-tool>`.
```

Comment:
Instead of using `topics-network-tool` here, I think it would be better if you added a line with something like `.. _requests-from-curl::` right above the new documentation you added to the network tool topic, and replaced `topics-network-tool` with `requests-from-curl` here. That should make this link to the specific paragraphs that cover copying requests in cURL format.